webapp/src/main/webapp/vue-apps/site-details/components/SiteNavigationTree.vue

```diff
@@ -119,6 +119,15 @@ export default {
           if (n.children?.length
               || n.pageLink
               || n.pageKey) {
+            if (!eXo.env.portal.documentsTrashNavigationEnabled) {
```

Comment:
this should not be done in social since it is specific to eXo

Reply:
This is a temporary change , it will be removed by removing the feature flag , **see PR description!**

Reply:
agree with @azayati